doc/releases/changelog-0.20.0.md

```diff
@@ -374,19 +401,103 @@
+
+* Added the `qml.QubitDensityMatrix` initialization gate for
+  mixed state simulation.
+  [(#1850)](https://github.com/PennyLaneAI/pennylane/pull/1850)
+
+* A thermal relaxation channel is added to the Noisy channels. The channel description can be
+  found on the supplementary information of [Quantum classifier with tailored quantum kernels](https://arxiv.org/abs/1909.02611).
+  [(#1766)](https://github.com/PennyLaneAI/pennylane/pull/1766)
+
+<h4>Manipulate QNodes to your ❤️s content with new transforms</h4>
 
-* The `merge_amplitude_embedding` transformation has been created to automatically merge all gates of this type into one.
+* The `merge_amplitude_embedding` transformation has been created to
+  automatically merge all gates of this type into one.
   [(#1933)](https://github.com/PennyLaneAI/pennylane/pull/1933)
+  ```python
+  from pennylane.transforms import merge_amplitude_embedding
+
+  @merge_amplitude_embedding
+  def qfunc():
 
-* The `undo_swaps` transformation has been created to automatically remove all swaps of a circuit.
+      qml.AmplitudeEmbedding([0,1,0,0], wires = [0,1])
+      qml.AmplitudeEmbedding([0,1], wires = 2)
+
+      return qml.expval(qml.PauliZ(wires = 0))
+
+  dev = qml.device("default.qubit", wires = 3)
+  qnode = qml.QNode(qfunc, dev)
+  print(qml.draw(qnode)())
+  ```
+  ```pycon
+   0: ──╭AmplitudeEmbedding(M0)──┤ ⟨Z⟩
+   1: ──├AmplitudeEmbedding(M0)──┤
+   2: ──╰AmplitudeEmbedding(M0)──┤
+   M0 =
+   [0.+0.j 0.+0.j 0.+0.j 1.+0.j 0.+0.j 0.+0.j 0.+0.j 0.+0.j]
+  ```
+
+* The `undo_swaps` transformation has been created to automatically remove all
+  swaps of a circuit.
   [(#1960)](https://github.com/PennyLaneAI/pennylane/pull/1960)
+  ```python
+  dev = qml.device('default.qubit', wires=3)
+
+  @qml.qnode(dev)
+  @qml.transforms.undo_swaps
+  def qfunc():
+      qml.Hadamard(wires=0)
+      qml.PauliX(wires=1)
+      qml.SWAP(wires=[0,1])
+      qml.SWAP(wires=[0,2])
+      qml.PauliY(wires=0)
+      return qml.expval(qml.PauliZ(0))
+
+  print(qml.draw(qfunc)())
+  ```
+  ```pycon
+   0: ──Y──┤ ⟨Z⟩
+   1: ──H──┤
+   2: ──X──┤
+  ```
 
 <h3>Improvements</h3>
 
+* Added functions for computing the values of atomic and molecular orbitals at a given position.
+  [(#1867)](https://github.com/PennyLaneAI/pennylane/pull/1867)
+
+  The functions `atomic_orbital` and `molecular_orbital` can be used, as shown in the
+  following codeblock, to evaluate the orbitals. By generating values of the orbitals at different
+  positions, one can plot the spatial shape of a desired orbital.
+
+  ```python
+  symbols  = ['H', 'H']
+  geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]], requires_grad = False)
+  mol = hf.Molecule(symbols, geometry)
+  hf.generate_scf(mol)()
+
+  ao = mol.atomic_orbital(0)
+  mo = mol.molecular_orbital(1)
+  ```
+
+  ```pycon
+  >>> print(ao(0.0, 0.0, 0.0))
+  >>> print(mo(0.0, 0.0, 0.0))
+  0.6282468778183719
+  0.018251285973461928
+  ```
+
+* Added support for Python 3.10.
```

Comment:
@josh146 it seems that TensorFlow doesn't have binaries for Python 3.10. Is it worth mentioning that here? We don't depend on TF by default, so maybe not relevant? :thinking:

Reply:
yep, I wouldn't worry about this :)